resources/keywords/create_provider_account.resource

```diff
@@ -3,17 +3,16 @@ Resource    ../keywords/operators.resource
 Resource    ../object_repo/database_access_obj.resource
 Resource    ../object_repo/installed_operators_obj.resource
 Library     ../../utils/scripts/util.py
+Library     ../../utils/scripts/dbaas_connection.py
 
 
 *** Variables ***
-${PROJECT}          dedicated-admin
 ${provaccname}      ${EMPTY}
 
-
 *** Keywords ***
 User Filters Project Other Than redhat-dbaas-operator And Navigates To Database Access Page
     User Navigates To Installed Operators Under Operators
-    User Filters ${PROJECT} Namespace On Project Dropdown
+    User Filters dedicated-admin Namespace On Project Dropdown
```

Comment:
Why hard coded?

Reply:
The keyword is parameterized (below screenshot) and while calling out I have directly used the value instead of referring another variable/parameter.
![Screenshot from 2022-05-11 22-22-08](https://user-images.githubusercontent.com/86094767/167904682-4e26ab68-692e-4617-a0ad-b569ffff9722.png)